tests/pytest.ini

```diff
@@ -1,2 +1,4 @@
 [pytest]
 addopts=-n 35
+markers =
```

Comment:
Wondering if we should have 
```

[pytest]
addopts=-n 1

```

Reply:
Hmmm, addopts=-n 35 only applies when we directly run the tests ourselves, so if it fails we can rerun it (unlike in CI/CD) => it speeds up the tests quite a lot.. I think we should keep it (since we already override it in ci/cd)